src/components/Accordion/sgds-accordion-item.ts

```diff
@@ -19,6 +19,12 @@ import styles from "./sgds-accordion-item.scss";
  * @csspart base - The accordion-item base wrapper
  * @csspart header - The accordion-item button header
  * @csspart content - The accordion-item content
+ *
+ * @cssprop --accordion-item-padding-y - The top and bottom padding for the container of accordion item's content
+ * @cssprop --accordion-item-padding-x - The right and left padding for the container of accordion item's content
+ * @cssprop --accordion-item-border-radius - The border radius of the accordion item
+ * @cssprop --accordion-item-font-weight - The font weight of accordion-button when it is not collapsed
+ * @cssprop --accordion-item-line-height- The line height of accordion
```

Comment:
small fix here. storybook is showing `--accordion-item-line-height-`

```suggestion
 * @cssprop --accordion-item-line-height - The line height of accordion
```